src/js/view/preview.js

```diff
@@ -1,13 +1,53 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, utils) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
+        this.model.on('change:mediaModel', onMediaModel, this);
```

Comment:
We need to do this because mediaModel changes when we get a new playlist item, so if we just listen to this.model.mediaModel.on('change:mediaType'), we do not get anything.